app/controllers/admin/commands_controller.rb

```diff
@@ -1,5 +1,8 @@
 class Admin::CommandsController < ApplicationController
-  before_action :authorize_admin!
+  include ProjectLevelAuthorization
```

Comment:
Ah, can't seem to do that, seems the before_actions override each other in some weird way. When I tried the above, the before_action for the :index, :create, :destroy was removed and let anyone through.